source-repo-scripts/source_changelog.bash

```diff
@@ -1,23 +1,17 @@
 #!/bin/bash
 # bash source_changelog.bash <PREV_VER>
 
-PREV_TAG=$1
+PREV_VER=$1
 
 git fetch --tags
 
-REPO=$(basename `git rev-parse --show-toplevel`)
```

Comment:
We can not remove REPO since it is being used to display the URLS in the Changelog. https://github.com/gazebo-tooling/release-tools/pull/826/files#diff-7d0c944be1fca9afa65ac904588d4ae5f5ade82c98276d72ebf6eaa3c4a43ac5R24

Reply:
oops!!! thanks for catching that, should be fixed in https://github.com/gazebo-tooling/release-tools/pull/826/commits/cd87e3c7c02205e623088467ed41497fb5794eee